airflow/providers/cncf/kubernetes/operators/spark_kubernetes.py

```diff
@@ -59,7 +59,7 @@ class SparkKubernetesOperator(BaseOperator):
     def __init__(
         self,
         *,
-        application_file: str | dict,
```

Comment:
Just curious, would users have passed `application_file` as a `dict` before or did it just not work previously? I'm just thinking of any backwards compat issues here albeit potentially small.

Reply:
Probably yes then